src/components/ContactDetails/ContactDetails.tsx

```diff
@@ -103,7 +103,9 @@ const ContactDetails: React.FC<ContactDetailsProps> = props => {
     <>
       <NavBar
         type='back'
-        onClick={() => props.history.push(ROUTES.BUDDY_TASKS_LIST)}
+        onClick={() =>
+          props.history.push(ROUTES.BUDDY_TASKS_LIST.replace(':newbieId', newbieId))
```

Comment:
I belive better to declare it as a separate function: [Avoid anonymous functions in render](https://medium.com/@User3141592/react-gotchas-and-best-practices-2d47fd67dd22#3f77)